### PR TITLE
Fix test failures after correction of MGX-1220

### PR DESCRIPTION
**Pull request by AlexChetverov**

### CODE REVIEW

File: test/sequential/maintenance-pallet.api.rolldown.test.ts [hunk 1]
```diff
@@ -236,7 +236,13 @@ describe.each(["mm", "upgradabilityMm"])(
...
         async (testName) => {
           testLog.getLog().info("DEBUG::TestName - " + testName);
-          const [extrinsic, signer] = tests[testName];
+          let signer: User;
+          const [extrinsic, testsSigner] = tests[testName];
+          if (mmMode === "mm") {
+            signer = testsSigner;
+          } else {
+            signer = users[3];
+          }
```
Comment (goncer) [inline test/sequential/maintenance-pallet.api.rolldown.test.ts:245]: why not simply creating a new user, this looks quite strange. isn't it?

File: test/sequential/sequencerStaking.cancel.test.ts [hunk 2]
```diff
@@ -363,12 +363,20 @@ it("GIVEN a sequencer, WHEN <in-correctly> canceling an update AND cancelator pr
...
+        chain,
+        "StakeOnly",
+      ),
+    ),
   );
   await testUser2.refreshAmounts(AssetWallet.BEFORE);
   await waitSudoOperationSuccess(cancelResolutionEvents, "SudoAsDone");
   await waitForNBlocks(disputePeriodLength + 1);
   await testUser2.refreshAmounts(AssetWallet.AFTER);
+  await Sudo.asSudoFinalized(Sudo.sudoAs(testUser2, providingExtrinsic));
```
Comment (goncer) [inline test/sequential/sequencerStaking.cancel.test.ts:379]: why the user needs to rejoin? 
 + rename the "providingExtrinsic" by "stakeAndJoinExtrinsic" to match the real behavior.

File: utils/rollDown/SequencerStaking.ts [hunk 3]
```diff
@@ -35,6 +36,8 @@ export class SequencerStaking {
       chainName,
       stringToBN(amountToStake.toString()),
       null,
+      //@ts-ignore
+      stakeAction,
```
Comment (goncer) [inline utils/rollDown/SequencerStaking.ts:40]: ask for new types, install them, and remove the ts-ignore.

**Review by goncer — CHANGES_REQUESTED** [review]

File: test/sequential/sequencerStaking.cancel.test.ts (hunk 2 above)
Comment (AlexChetverov) [reply test/sequential/sequencerStaking.cancel.test.ts:379]: It's my mistake, deleted. Variable was renamed

File: test/sequential/maintenance-pallet.api.rolldown.test.ts (hunk 1 above)
Comment (AlexChetverov) [reply test/sequential/maintenance-pallet.api.rolldown.test.ts:245]: I decided that if we have an unused account, we can use it here. But yes, we can create a new user. Added

File: utils/rollDown/SequencerStaking.ts (hunk 3 above)
Comment (AlexChetverov) [reply utils/rollDown/SequencerStaking.ts]: Done

**Review by goncer — APPROVED** [review]

**Pull request merged**